crates/holochain/tests/dna_properties/mod.rs

```diff
@@ -0,0 +1,89 @@
+use hdk::prelude::*;
+use holochain::sweettest::SweetAgents;
+use holochain::sweettest::SweetConductor;
+use holochain::sweettest::SweetDnaFile;
+use holochain_conductor_api::conductor::ConductorConfig;
+use holochain_wasm_test_utils::TestWasm;
+use holochain_zome_types::DnaModifiersOpt;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Debug, Eq, PartialEq, SerializedBytes, Clone)]
+pub struct MyValidProperties {
+    authority_agent: Vec<u8>,
+    max_count: u32,
+    contract_address: String,
+}
```

Comment:
Can you import this struct from the test wasm? https://github.com/holochain/holochain/pull/2880/files#diff-3d763c77d786763ca68bebd50478414f9acca8f4fc9e3e5799a00859e7f30f97R4-R8

Reply:
done